templates/agent-tls-template.yml

```diff
@@ -326,9 +326,11 @@ objects:
         - name: mqtt-certs
           secret:
             secretName: maestro-agent-certs
+            optional: true
```

Comment:
why do you make it optional? for agent-tls-template, they are required.

Reply:
the agent need to support grpc/mqtt broker, if use grpc broker, this is not needed